R/calcFeDemandIndustry.R

```diff
@@ -5,6 +5,8 @@
 #'   to `FALSE`.)
 #' @param last_empirical_year Last year for which empirical data is available.
 #'   Defaults to 2020.
+#' @param scenario String designating the scenario. !!Currently it acts as a filter only, with the actual scenarios
+#' computed within the function hard-coded into remind_scenarios.
```

Comment:
This is not a sting designating one scenario, but a character vector designating several.  Should be named and documented accordingly.